### PR TITLE
[Docs]Batch04 - Bitwise numeric functions

### PR DESCRIPTION
**Pull request by edgar2020**
### Description

Add examples to the sql-functions.md page.

Updates the BITWISE_AND, BITWISE_COMPLEMENT, BITWISE_CONVERT_DOUBLE_TO_LONG_BITS, BITWISE_CONVERT_LONG_BITS_TO_DOUBLE, BITWISE_OR, BITWISE_SHIFT_LEFT, BITWISE_SHIFT_RIGHT, and BITWISE_XOR.

This PR has:

- [X] been self-reviewed.

### CODE REVIEW

File: docs/querying/sql-functions.md [hunk 1]
```diff
@@ -336,68 +336,189 @@ Performs a bitwise XOR operation on all input values.
...
-`BITWISE_SHIFT_LEFT(expr1, expr2)`
+Returns the bitwise left shift by x positions of an expr, that is, `expr << x`.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `BITWISE_SHIFT_LEFT(expr, x)`
+* **Function type:** Scalar, numeric
+
+<details><summary>Example</summary>
+
+The following example performs the bitwise SHIFT operation `2 << 3`.
```
Comment (edgar2020) [inline docs/querying/sql-functions.md:463]: BITWISE_SHIFT_LEFT and BITWISE_SHIFT_RIGHT did not mention what the second parameter did, so I tried to include that in the definition, not sure how the working of the definition can be further improved.

File: docs/querying/sql-functions.md [hunk 2]
```diff
@@ -336,68 +336,189 @@ Performs a bitwise XOR operation on all input values.
...
+| -- |
+| -13 | 
+</details>
+
+[Learn more](sql-scalar.md#numeric-functions)
 
 ## BITWISE_CONVERT_DOUBLE_TO_LONG_BITS
 
-`BITWISE_CONVERT_DOUBLE_TO_LONG_BITS(expr)`
+Converts the bits of an IEEE 754 floating-point double value to long.
```
Comment (edgar2020) [inline docs/querying/sql-functions.md:386]: The definition for BITWISE_CONVERT_DOUBLE_TO_LONG_BITS and BITWISE_CONVERT_LONG_BITS_TO_DOUBLE  is a little hard to follow, especially with no context of binary are the different binary formats. Would like feedback on if this enough of an explanation given it is just a reference guide.

File: docs/querying/sql-functions.md [hunk 3]
```diff
@@ -336,68 +336,189 @@ Performs a bitwise XOR operation on all input values.
...
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:**`BITWISE_CONVERT_LONG_BITS_TO_DOUBLE(expr)`
+* **Function type:** Scalar, numeric
 
-Converts a long to the IEEE 754 floating-point double specified by the bits stored in the long.
+<details><summary>Example</summary>
 
-## BITWISE_OR
+The following example returns the long representation of `4643176031446892544` as an IEEE 754 floating-point double.
```
Comment (asdf2014) [inline docs/querying/sql-functions.md:416]:
```suggestion
The following example returns the long representation of `4643176031446892544` as an IEEE 754 floating-point double.
```

File: docs/querying/sql-functions.md (hunk 2 above)
Comment (techdocsmith) [reply docs/querying/sql-functions.md:386]: I think it is fine as long as they match the definitions in the Scalar Functions list.

File: docs/querying/sql-functions.md (hunk 1 above)
Comment (techdocsmith) [reply docs/querying/sql-functions.md:463]: I think your description is correct. Binary is a positional notation. This shifts the numbers `n` positions to the left.

**Review by techdocsmith — CHANGES_REQUESTED** [review]
Great job. minor suggestions mainly about using a colon (:) notation instead of repeating "that is" every time.

File: docs/querying/sql-functions.md [hunk 4]
```diff
@@ -336,68 +336,189 @@ Performs a bitwise XOR operation on all input values.
 
 ## BITWISE_AND
 
-`BITWISE_AND(expr1, expr2)`
+Returns the bitwise AND between two expressions, that is, `expr1 & expr2`.
```
[the remark this answers is not in the document]
Comment (edgar2020) [reply docs/querying/sql-functions.md]: applied it manually

**Review by techdocsmith — APPROVED** [review]
LGTM
